selfdrive/car/hyundai/carstate.py

```diff
@@ -99,10 +99,11 @@ def update(self, cp, cp_cam):
       ret.cruiseState.enabled = cp.vl["TCS13"]["ACC_REQ"] == 1
       ret.cruiseState.standstill = False
     else:
-      ret.cruiseState.available = cp_cruise.vl["SCC11"]["MainMode_ACC"] == 1
+      scc_bus = "SCC12" if self.CP.flags & HyundaiFlags.CAN_CANFD.value else "SCC11"
+      ret.cruiseState.available = cp_cruise.vl[scc_bus]["MainMode_ACC"] == 1
       ret.cruiseState.enabled = cp_cruise.vl["SCC12"]["ACCMode"] != 0
```

Comment:
is this hard coded on purpose to 12?

Reply:
Yes, these cars have the signals on `SCC12`.

Reply:
thanks.  to be clear, i meant most of the stanza was changed from a static `SCC12` to the variable `scc_bus`, but line 104 is left with `SCC12` string instead of the new variable.

but looking at it now i guess this particular signal just happens to be in the same place on both models